coffee-shop/src/pages/Order/components/QuantityOrdered/QuantityOrdered.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import styles from "./QuantityOrdered.module.css";
+import QuantityButton from "../QuantityButton";
+import icons from "../../../../assets/icons/iconImport";
+
+const QuantityOrdered = ({ image, name }) => {
```

Comment:
great component, but how can you read the quantity outside the component, you need function in props to read the quantity from parent of component

Reply:
Done